srf_generation/source_parameter_generation/uncertainties/versions/V19p8.py

```diff
@@ -20,7 +20,9 @@
 
 
 def generate_source_params(
-    sources_line: Union[GCMT_Source, NHM_Source]
+    sources_line: Union[GCMT_Source, NHM_Source],
+    additional_source_parameters: Dict[str, Any],
+    **kwargs,
```

Comment:
probably a good idea to add a logger for precocious, so user can track what parameters are being used and what is not. ( dont wanna burn hundred thousands of core-hours just to realize the parameters are not being used)